components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -400,13 +401,18 @@ export class ReportingSearchbarComponent implements OnInit {
     this.filterRemoved.emit({ detail: filter });
   }
 
+  // month - month with January 0 and December 11
+  // year - 4 digit year (2019)
   onMonthSelect([month, year]) {
-    this.visibleDate.setMonth(month);
-    this.visibleDate.setFullYear(year);
+    this.visibleDate.month(month);
+    this.visibleDate.year(year);
   }
 
   onDaySelect(date: string) {
-    this.visibleDate = new Date(date);
+    const m = moment.utc(date);
+    this.visibleDate.date(m.date());
+    this.visibleDate.month(m.month());
+    this.visibleDate.year(m.year());
```

Comment:
Only changing the date, month, and year. Trying not to update the hour to complicate things with time zones.